src/components/ArticleVotes.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react"
+import { patchArticleById } from '../api'
+
+const ArticleVotes = ({article_id, votes}) => {
+
+    const [votesChange, setVotes]= useState(0)
+
+    function incVote(increment) {
+        setVotes((currVote) => 
+        currVote + increment);
+        patchArticleById(article_id, increment)
+        .catch(() => {
```

Comment:
It would be good to let the user know with some sort of error message if the vote fails